### PR TITLE
[keyboard] Wabi handwire

### PR DESCRIPTION
**Pull request by Rossman360**
## Description

Thought I added this ages ago but had to recreate.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ x] Keyboard (addition or update)
- [ x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/wabi/rules.mk [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# MCU name
+MCU = atmega32u4
+# Bootloader selection
+#   Teensy halfkay Pro Micro caterina Atmel DFU atmel-dfu LUFA DFU lufa-dfu
+#   QMK DFU qmk-dfu ATmega32A bootloadHID ATmega328P USBasp
+BOOTLOADER = atmel-dfu
```
Comment (zvecr) [inline keyboards/handwired/wabi/rules.mk:6]:
```suggestion

# Bootloader selection
BOOTLOADER = atmel-dfu

# Build Options
#   change yes to no to disable
#
```

File: keyboards/handwired/wabi/config.h [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0xB07D
+#define DEVICE_VER 0x0002
+#define MANUFACTURER LifeIsOnTheWire
+#define PRODUCT PostageBoard
+#define DESCRIPTION "A Handwire Controller Board Round 2"
```
Comment (fauxpark) [inline keyboards/handwired/wabi/config.h:26]:
```suggestion
```

File: keyboards/handwired/wabi/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+#define _QWERTY 0
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_QWERTY] = LAYOUT(
+    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, \
+    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, \
+    KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT, \
+    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP, \
+    KC_LCTL, KC_LGUI,          KC_LALT, KC_SPC,  KC_PGDN,  KC_ENT,  KC_SPC,  KC_BSPC,                   KC_LEFT, KC_DOWN, KC_RGHT  \
```
Comment (fauxpark) [inline keyboards/handwired/wabi/keymaps/default/keymap.c:12]:
```suggestion
    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
    KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,
    KC_LCTL, KC_LGUI,          KC_LALT, KC_SPC,  KC_PGDN,  KC_ENT,  KC_SPC,  KC_BSPC,                   KC_LEFT, KC_DOWN, KC_RGHT
```

File: keyboards/handwired/wabi/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,14 @@
+#include QMK_KEYBOARD_H
+
+#define _QWERTY 0
+
+
```
Comment (fauxpark) [inline keyboards/handwired/wabi/keymaps/default/keymap.c:5]:
```suggestion
```

File: keyboards/handwired/wabi/keymaps/rossman360/keymap.c [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+    _______, _______,          _______, PARADOWN, CSPEAK, _______, _______, _______,                   _______, _______, _______  \
+  ),
+[_NUM] = LAYOUT(
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,\
+    _______, _______, _______, _______, _______, _______, _______, KC_7   , KC_8   , KC_9   , _______, _______, _______, _______, \
+    _______, _______, _______, _______, _______, _______, KC_MINS, KC_4   , KC_5   , KC_6   , KC_COLN, _______, _______, \
+    _______, _______, _______, _______, _______, _______, _______, KC_1   , KC_2   , KC_3   , KC_BSLS, _______, _______, \
+    _______, _______,          _______, KC_0   , _______, KC_NO  , KC_SPC , KC_0   ,                   _______, _______, _______  \
```
Comment (fauxpark) [inline keyboards/handwired/wabi/keymaps/rossman360/keymap.c:52]:
```suggestion
    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
    JUMPBACK,KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, RSMOD  , KC_UP,
    CMDBSP,  ALTDEL,           CTRLSP,  TABMOD,  PGMOD  , ENTMOD , SPCMOD , MO(_DEL),                  KC_LEFT, KC_DOWN, KC_RGHT
  ),
[_DEL] = LAYOUT(
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    REMCAPS, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______,          BLINE  , KC_BSPC, BWORD  , KC_NO  , KC_NO  , _______,                   _______, _______, _______
  ),
[_FN1] = LAYOUT(
    RESET  , _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, NTAB,
    _______, _______, _______, _______, _______, _______, _______, UNDO   , _______, _______, _______, _______, _______, KC_SLSH,
    _______, _______, _______, _______, _______, _______, KC_HOME, KC_LEFT, KC_UP  , KC_RIGHT,KC_END , _______, CTAB,
    _______, _______, XPANDR , _______, PMERGE , _______, _______, PMERGE , KC_DOWN, _______, _______, _______, _______,
    _______, _______,          _______, LWORD  , RWORD  , KC_NO  , _______, KC_NO  ,                   _______, _______, _______
  ),
[_FN2] = LAYOUT(
    _______, SPEAK1 , SPEAK2 , SPEAK3 , _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______,          _______, PARADOWN, CSPEAK, _______, _______, _______,                   _______, _______, _______
  ),
[_NUM] = LAYOUT(
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, KC_7   , KC_8   , KC_9   , _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, KC_MINS, KC_4   , KC_5   , KC_6   , KC_COLN, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, KC_1   , KC_2   , KC_3   , KC_BSLS, _______, _______,
    _______, _______,          _______, KC_0   , _______, KC_NO  , KC_SPC , KC_0   ,                   _______, _______, _______
```

File: keyboards/handwired/wabi/readme.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+* Keyboard Maintainer: [Rossman360](https://github.com/rossman360)
+* Hardware Supported: Postage Board Mini
+* Hardware Availability: [Montsinger.net](https://montsinger.net)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make handwired/wabi:default
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start$
+
+
```
Comment (fauxpark) [inline keyboards/handwired/wabi/readme.md:16]:
```suggestion

Flashing example for this keyboard:

    make %KEYBOARD%:default:flash

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/handwired/wabi/rules.mk [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+SLEEP_LED_ENABLE = no # Breathing sleep LED during USB suspend
+# if this doesn't work, see here:https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no # USB Nkey Rollover
+BACKLIGHT_ENABLE = no # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no # Enable keyboard RGB underglow
+MIDI_ENABLE = no # MIDI support
+BLUETOOTH_ENABLE = no # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no # Audio output on port C6
+FAUXCLICKY_ENABLE = no # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no # Enable support for HD44780
```
Comment (fauxpark) [inline keyboards/handwired/wabi/rules.mk:22]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here:https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/handwired/wabi/config.h [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0xB07D
+#define DEVICE_VER 0x0002
+#define MANUFACTURER LifeIsOnTheWire
+#define PRODUCT PostageBoard
```
Comment (fauxpark) [inline keyboards/handwired/wabi/config.h:25]: You should probably change these.

File: keyboards/handwired/wabi/readme.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+![Wabi](https://i.imgur.com/cCyv5jF.jpg)
+
+A conjoined-split 60% ortholinear amoeba handwire made by Ross Montsinger with 104-keycap sets in mind.
+
+* Keyboard Maintainer: [Rossman360](https://github.com/rossman360)
+* Hardware Supported: Postage Board Mini
+* Hardware Availability: [Montsinger.net](https://montsinger.net)
+
+Flashing example for this keyboard:
```
Comment (fauxpark) [inline keyboards/handwired/wabi/readme.md:11]:
```suggestion
Make example for this keyboard (after setting up your build environment):

    make handwired/wabi:default

Flashing example for this keyboard:
```

File: keyboards/handwired/wabi/info.json [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
..."y":-1.0}, {"x":0, "y":0, "w":1.25}, {"x":1.25, "y":0, "w":1.25}, {"x":4, "y":0, "w":1.25}, {"x":5.25, "y":0}, {"x":6.25, "y":0}, {"label":"6", "x":-8, "y":1}, {"label":"7", "x":-7.0, "y":1}, {"label":"8", "x":-6, "y":1}, {"label":"9", "x":-5.0, "y":1}, {"label":"0", "x":-4, "y":1}, {"label":"-", "x":-3.0, "y":1}, {"label":"=", "x":-2, "y":1}, {"label":"Backspace", "x":-1.0, "y":1, "w":2}, {"label":"Y", "x":-8, "y":2}, {"label":"U", "x":-7.0, "y":2}, {"label":"I", "x":-6, "y":2}, {"label":"O", "x":-5.0, "y":2}, {"label":"P", "x":-4, "y":2}, {"label":"[", "x":-3.0, "y":2}, {"label":"]", "x":-2, "y":2}, {"label":"|", "x":-1.0, "y":2, "w":1.5}, {"label":"H", "x":-8, "y":3}, {"label":"J", "x":-7.0, "y":3}, {"label":"K", "x":-6, "y":3}, {"label":"L", "x":-5.0, "y":3}, {"label":";", "x":-4, "y":3}, {"label":"'", "x":-3.0, "y":3}, {"label":"Enter", "x":-2, "y":3, "w":2.25}, {"label":"N", "x":-8, "y":4}, {"label":"M", "x":-7.0, "y":4}, {"label":",", "x":-6, "y":4}, {"label":".", "x":-5.0, "y":4}, {"label":"?", "x":-4, "y":4}, {"label":"Shift", "x":-3.0, "y":4, "w":2}, {"x":-1.0, "y":4}, {"x":-8, "y":5}, {"x":-7.0, "y":5}, {"x":-6, "y":5}, {"x":-2, "y":5}, {"x":-1.0, "y":5}, {"x":0, "y":5}]
```
Comment (noroadsleft) [inline keyboards/handwired/wabi/info.json:9]: This is completely messed up:

image

```suggestion
            "layout": [{"label":"Esc", "x":1.25, "y":0}, {"label":"1", "x":2.25, "y":0}, {"label":"2", "x":3.25, "y":0}, {"label":"3", "x":4.25, "y":0}, {"label":"4", "x":5.25, "y":0}, {"label":"5", "x":6.25, "y":0}, {"label":"6", "x":8.75, "y":0}, {"label":"7", "x":9.75, "y":0}, {"label":"8", "x":10.75, "y":0}, {"label":"9", "x":11.75, "y":0}, {"label":"0", "x":12.75, "y":0}, {"label":"-", "x":13.75, "y":0}, {"label":"=", "x":14.75, "y":0}, {"label":"Backspace", "x":15.75, "y":0, "w":2}, {"label":"Tab", "x":0.75, "y":1, "w":1.5}, {"label":"Q", "x":2.25, "y":1}, {"label":"W", "x":3.25, "y":1}, {"label":"E", "x":4.25, "y":1}, {"label":"R", "x":5.25, "y":1}, {"label":"T", "x":6.25, "y":1}, {"label":"Y", "x":8.75, "y":1}, {"label":"U", "x":9.75, "y":1}, {"label":"I", "x":10.75, "y":1}, {"label":"O", "x":11.75, "y":1}, {"label":"P", "x":12.75, "y":1}, {"label":"[", "x":13.75, "y":1}, {"label":"]", "x":14.75, "y":1}, {"label":"\\", "x":15.75, "y":1, "w":1.5}, {"label":"Caps Lock", "x":0.5, "y":2, "w":1.75}, {"label":"A", "x":2.25, "y":2}, {"label":"S", "x":3.25, "y":2}, {"label":"D", "x":4.25, "y":2}, {"label":"F", "x":5.25, "y":2}, {"label":"G", "x":6.25, "y":2}, {"label":"H", "x":8.75, "y":2}, {"label":"J", "x":9.75, "y":2}, {"label":"K", "x":10.75, "y":2}, {"label":"L", "x":11.75, "y":2}, {"label":";", "x":12.75, "y":2}, {"label":"'", "x":13.75, "y":2}, {"label":"Enter", "x":14.75, "y":2, "w":2.25}, {"label":"Shift", "x":0, "y":3, "w":2.25}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":8.75, "y":3}, {"label":"M", "x":9.75, "y":3}, {"label":",", "x":10.75, "y":3}, {"label":".", "x":11.75, "y":3}, {"label":"/", "x":12.75, "y":3}, {"label":"Shift", "x":13.75, "y":3, "w":2.75}, {"label":"Up", "x":16.5, "y":3}, {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4, "w":1.25}, {"label":"Alt", "x":4, "y":4, "w":1.25}, {"label":"Space", "x":5.25, "y":4}, {"label":"Page Down", "x":6.25, "y":4, "w":1.25}, {"label":"Enter", "x":8.5, "y":4, "w":1.25}, {"label":"Space", "x":9.75, "y":4}, {"label":"Backspace", "x":10.75, "y":4, "w":1.25}, {"label":"Left", "x":15.5, "y":4}, {"label":"Down", "x":16.5, "y":4}, {"label":"Right", "x":17.5, "y":4}]
```

File: keyboards/handwired/wabi/readme.md [hunk 11]
```diff
@@ -0,0 +1,19 @@
+# Wabi
+
+![Wabi](https://i.imgur.com/cCyv5jF.jpg)
```
Comment (noroadsleft) [inline keyboards/handwired/wabi/readme.md:3]: Swaps this image for a smaller version.

```suggestion
![Wabi](https://i.imgur.com/cCyv5jFl.jpg)
```

File: keyboards/handwired/wabi/rules.mk [hunk 12]
```diff
@@ -0,0 +1,23 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTLOADER = atmel-dfu
```
Comment (noroadsleft) [inline keyboards/handwired/wabi/rules.mk:10]: Already defined on Line 5.

```suggestion
```

File: keyboards/handwired/wabi/info.json [hunk 13]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "Wabi", 
+    "url": "www.montsinger.net", 
+    "maintainer": "Montsinger", 
+    "width": 7.25, 
+    "height": 6,
```
Comment (noroadsleft) [inline keyboards/handwired/wabi/info.json:6]:
```suggestion
    "width": 18.5,
    "height": 5,
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**